### DISCUSSION TITLE
xpack  在打包过程中能执行 target 的 before_package 类似过程吗

### DISCUSSION THREAD

**Discussion by StarBeat** [xmake-io/xmake#4653] [Q&A] [answered]
xpack  执行关联 target 的 build，如果能有 before_xpack 就好了

**Comment by waruqi** [comment]
不是原本就有么，仔细看文档 https://xmake.io/#/zh-cn/manual/xpack?id=xpackbefore_package

**Reply by StarBeat** [reply]
希望 taget 本身的 before_package，或者 rule 的 before_package 可以执行

**Reply by waruqi** [reply, marked as answer]
不考虑
